crates/bitwarden/src/crypto/mod.rs

```diff
@@ -112,6 +115,16 @@ pub(crate) fn stretch_key(secret: [u8; 16], name: &str, info: Option<&str>) -> S
     SymmetricCryptoKey::try_from(key.as_slice()).unwrap()
 }
 
+pub fn decrypt(cipher: &CipherString, key: &SymmetricCryptoKey) -> Result<Vec<u8>> {
```

Comment:
I think this should be moved into EncString?

Reply:
That sounds reasonable, though that might cause conflicts with the Decryptable trait because they have the same name.